omnia/lib/config.sh

```diff
@@ -80,6 +77,24 @@ importNetwork () {
 	export ETH_RPC_URL
 }
 
+importGasPrice () {
+	local _json="$1"
+
+	# Getting Gas price details
+	ETH_GAS_SOURCE="$(echo "$_json" | jq -S '.ethereum.gasPrice.source // "node"')"
```

Comment:
```suggestion
	ETH_GAS_SOURCE="$(echo "$_json" | jq -rS '.ethereum.gasPrice.source // "node"')"
```
You need this to strip quotes. Otherwise omnia will always use `node` as the gas price source.